test/groovy/com/sap/piper/versioning/DockerArtifactVersioningTest.groovy

```diff
@@ -20,7 +20,24 @@ class DockerArtifactVersioningTest extends BasePiperTest{
 
     String passedDir
 
-    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this, 'test/resources/versioning/DockerArtifactVersioning')
+    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this,
+        ['Dockerfile': ('''|FROM rootImage:1.2.3
```

Comment:
Might be hard to read/format. Why not keeping both options? I think for larger files, you still would like to store them as files.